server/server.go

```diff
@@ -53,6 +54,8 @@ import (
 	"github.com/prometheus/client_golang/prometheus/promhttp"
 )
 
+var Environment = os.Getenv("AGGREGATOR_ENV")
```

Comment:
I'd urge to use Configuration there, not "package global" variables. Please

Reply:
for example we can add "production bool" into the Configuration. Do the config setup properly in main package and let be server itself basically global-variable free. WDYT?